packages/mobile/src/components/drawer/Drawer.tsx

```diff
@@ -217,16 +219,17 @@ export const springToValue = ({
       // Factor the height of the drawer into the spring physics.
       // Without this, short drawers tend to feel sluggish while
       // tall drawers really get going.
-      tension = 70 + 60 * (1 - drawerHeight / FULL_DRAWER_HEIGHT)
-      friction = 10 + 2 * (1 - drawerHeight / FULL_DRAWER_HEIGHT)
+      tension = 70 + 60 * (1 - Math.min(drawerHeight / FULL_DRAWER_HEIGHT, 1))
+      friction = 10 + 2 * (1 - Math.min(drawerHeight / FULL_DRAWER_HEIGHT, 1))
       break
   }
   Animated.spring(animation, {
     toValue: value,
     tension,
     friction,
     useNativeDriver: true,
-    velocity
+    velocity,
```

Comment:
maybe we add a "animationConfig" prop for various things, but i think this is all good :)